templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -31,16 +31,42 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>Again very small comment - I think there a small typo, should be <code>../data/example_dataset</code>. Also, I don't think we need quotes around <code>"../data/example_dataset"</code> , since it's already displaying as code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>